src/components/v2v3/V2V3Project/ProjectDashboard/components/PayProjectCard/components/TokensPerEth.tsx

```diff
@@ -1,19 +1,23 @@
 import { Trans } from '@lingui/macro'
-import { useTokensPerEth } from 'components/v2v3/V2V3Project/ProjectDashboard/hooks/useTokensPerEth'
 import { V2V3CurrencyOption } from 'models/v2v3/currencyOption'
+import { V2V3_CURRENCY_ETH } from 'utils/v2v3/currency'
+import { useTokensPerEth } from '../../../hooks/useTokensPerEth'
 
 export const TokensPerEth = ({
   currencyAmount,
 }: {
   currencyAmount:
     | {
-        amount: number
+        amount: string
```

Comment:
pretty sure this was also getting passed a string, but maybe an `any` type made the compiler happy